domains/elib0.json

```
@@ -0,0 +1,9 @@
+{
+  "owner": {
+    "username": "elib0",
+    "email": "elijose.c@gmail.com"
+  },
+  "record": {
+    "TXT": "elib0.github.io"
```

If you are trying to connect your GitHub Pages website, change record type to `CNAME`
```suggestion
    "CNAME": "elib0.github.io"
```